docs/source/user_documentation/replica_dot_yaml_file.rst

```diff
@@ -90,8 +90,7 @@ Your initial replica file will look something like `this
          default:
            margin_of_error: 0.05
            confidence: 0.95
-           min_sample_size: 300
-           max_allowed_rows: 1500000
```

Comment:
I removed it just because in sample replica.yml file that is created by command `snowshu init` there is no such line. I can rollback if my decision is wrong